package.json

```
@@ -5,7 +5,7 @@
   "scripts": {
     "dev": "next dev",
     "build": "copy .env .env.local && next build",
-    "build:prod": "next build",
+    "build:prod": "next build && cp -r out/it/404/index.html out/it/404/index.txt out/404/ ",
```

```suggestion
    "build:prod": "next build && cp -f out/it/404/index.html out/it/404/index.txt out/404/ ",
```

`-r` usually stands for recursive, am I right? We don't need a recursive operation I guess.

---

Updated